### ISSUE TITLE
Looking to hand over maintainership

### ISSUE THREAD

**Issue by djc** [djc/rnc2rng#41]
I'd like to stop maintaining this package, since I don't really have a use for it.

@brechtm, @yarikoptic, @rly, @yurivict any interest in taking over?

**Comment by brechtm** [comment]
Unfortunately I'm not even managing to spend any time on citeproc-py which relies on rnc2rng, so I'm going to pass.

**Comment by rly** [comment]
Sorry, I have too many other projects on my plate, so I am going to pass as well.

**Comment by yurivict** [comment]
Sorry, I don't use ```rnc2rng```, I only maintain its port on FreeBSD and perhaps I've created some issues here but that's the extent of my involvement.

**Comment by yarikoptic** [comment]
A similar crutch with time here BUT given low number of open issues and relatively small size of the project, I wonder what maintainership would involve here and may be we could collectively "keep an eye" on it?

**Comment by djc** [comment]
Yeah, the workload has been really minimal -- for the past year, really the only work has to be to update things to update Python versions in CI/package metadata. There was a tiny regression in Dec 2020, but that was easily taken care off as well. So it's really not a lot of work to maintain this.
